src/components/MyKiva/BadgeInProgress.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+	<div>
+		<div
+			class="tw-pt-1 tw-pb-1.5 tw-flex tw-flex-col md:tw-flex-row tw-items-center tw-justify-left tw-gap-3"
+		>
+			<BadgeContainer :status="BADGE_IN_PROGRESS" :shape="getBadgeShape()" class="tw-z-1">
+				<img
```

Comment:
The dashed border is showing up inside the badge icon. It should show around the outside. Is this because of the maxheight override? Also, it would be helpful to add `BadgeModal` stories to cover the 5 different badge use cases, to ensure the badge icon looks correctly for each.

Related, this version of the `BadgeModal` requires there to be modal content to be complete compared to the mocks (modal without title, back button, etc.). Is that something you see as a follow-up ticket?

Reply:
Sounds good regarding the follow-up ticket. Please include a link to the mocks and bullets for the specific requirements in the mocks that are still needed.